src/kibana/components/filter_bar/lib/mapGeoBoundingBox.js

```diff
@@ -0,0 +1,21 @@
+define(function (require) {
+  var _ = require('lodash');
+  return function mapGeoBoundBoxProvider(Promise, courier) {
+    return function (filter) {
+      var key, value, topLeft, bottomRight, field;
+      if (filter.geo_bounding_box) {
+        return courier
+        .indexPatterns
+        .get(filter.meta.index).then(function (indexPattern) {
+          key = _.keys(filter.geo_bounding_box)[0];
+          field = indexPattern.fields.byName[key];
+          topLeft = field.format.convert(filter.geo_bounding_box[field.name].top_left);
+          bottomRight = field.format.convert(filter.geo_bounding_box[field.name].bottom_right);
+          value = topLeft + ' to ' + bottomRight;
```

Comment:
The nature of the bounding box means that these can sometimes be really high precision numbers. It feels like we shouldn't show -15.453453421567, but thats what the filter is.